
import "msg_error.proto";
import "msg_struct.proto";

package msg;

// character

    message enter_town_req
    {
        required string auth_token = 1;
        required string char_name = 2;
    }

    message enter_town_ack
    {
        required err_type err_code = 1;
        required v3angle pos = 2;
    }

    message enter_dungeon_req
    {
        required string auth_token = 1;
        required string char_name = 2;
        required int32 dungeon_mid = 3;
    }

    message enter_dungeon_ack
    {
        required err_type err_code = 1;
        required v3angle pos = 2;
    }

// end

